pkg/jobs/trigger_at.go

```diff
@@ -0,0 +1,89 @@
+package jobs
+
+import (
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// AtTrigger implements the @at trigger type. It schedules a job at a specified
+// time in the future.
+type AtTrigger struct {
+	typ  string
```

Comment:
Because `type` is a reserved keyword and `Type()` is a method is part of the `Trigger` interface